x/stakeibc/keeper/unbonding_records.go

```diff
@@ -242,68 +244,69 @@ func (k Keeper) InitiateAllHostZoneUnbondings(ctx sdk.Context, dayNumber uint64)
 	return success, successfulUnbondings, failedUnbondings
 }
 
+// Deletes any epoch unbonding records that have had all unbondings claimed
 func (k Keeper) CleanupEpochUnbondingRecords(ctx sdk.Context, epochNumber uint64) bool {
-	// this function goes through each EpochUnbondingRecord
-	// if all hostZoneUnbondings on an EpochUnbondingRecord's are processed, the EpochUnbondingRecord is deleted
+	k.Logger(ctx).Info("Cleaning Claimed Epoch Unbonding Records...")
+
 	for _, epochUnbondingRecord := range k.RecordsKeeper.GetAllEpochUnbondingRecord(ctx) {
-		k.Logger(ctx).Info(fmt.Sprintf("Cleaning up epoch unbondings for epoch unbonding record from epoch %d", epochUnbondingRecord.GetEpochNumber()))
 		shouldDeleteEpochUnbondingRecord := true
-		hostZoneUnbondings := epochUnbondingRecord.GetHostZoneUnbondings()
```

Comment:
Why switch from the `Get` method to direct access? Same question for several instances below e.g. `GetNativeTokenAmount()`

Reply:
chatted offline: TLDR, accessing fields directly is preferred over getters to prevent unintended consequences